samcli/cli/main.py

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
Can you instead add a __main__.py? `python -m ..` will call the __main__.py file - https://docs.python.org/3/library/__main__.html

Reply:
Moved, this was a good call out.